lighthouse/helpers/mysql_db.py

```diff
@@ -1,6 +1,6 @@
-import sqlalchemy  # type: ignore
-from sqlalchemy import MetaData  # type: ignore
-from sqlalchemy.engine.base import Engine  # type: ignore
+import sqlalchemy
```

Comment:
When do or don't you need that type: ignore comment? Not clear to me...

Reply:
I've added a `mypy.ini` config file which allows us to configure mypy and ignore packages which do not yet have [stub files](https://mypy.readthedocs.io/en/stable/stubs.html) in typeshep. There are edge cases where it will be easier to use `type: ignore` to tell mypy to ignore that line but it's usually worth the effort to resolve the type issues early on.